jwst/assign_wcs/util.py

```diff
@@ -210,6 +220,8 @@ def wcs_from_footprints(dmodels, refmodel=None, transform=None, bounding_box=Non
     bounding_box : tuple, optional
         Bounding_box of the new WCS.
         If not supplied it is computed from the bounding_box of all inputs.
+    pscale_ratio : float, optional
+        A scale representing the ratio of pixel scale of input to output.
```

Comment:
I think wording "... ratio of input to output pixel scale" would be a little more straightforward for a reader to parse.

Reply:
Fixed here and elsewhere.  Yes, clearer.